src/sentry/search/events/fields.py

```diff
@@ -1639,6 +1639,20 @@ def is_accessible(self, acl=None):
             default_result_type="number",
             redundant_grouping=True,
         ),
+        Function(
+            "cov",
```

Comment:
covarSamp is covariance right? I see a few places from looking this up its abbreviated to `cov`, but should we call it `covariance` to avoid confusion?

Reply:
I don't mind updating it to `covariance` it just deviates from the abbreviations we use for all the other function names like `avg` for `average`.

Reply:
hmmm i guess anyone who knows how to use covariance will probably be aware what cov is, lets leave it as is